docs/GETTING_STARTED.md

```diff
@@ -31,16 +31,19 @@ Installation information can be found in [SETUP.md](SETUP.md).
 
 When ecChronos is started, it will read repair history to see the status of the repairs.
 If repair history exists for a table, ecChronos will use the last time repair was run and add the repair interval to calculate the next repair time.
-On the other hand, if there's no repair history present for a table, the ecChronos will assume that the table has been repaired in the past.
+On the other hand, if there's no repair history present for a table, the ecChronos will assume that the table has been repaired in the past using 
+initial delay to calculate.
 
 The assumption is done in the following way:
+* Initial delay = must be lower than repair interval, configurable via ecc.yml
 * Repair delay in days = min(repair interval, 1)
-* Completed at = (start time - repair interval + repair delay)
+* Completed at = (start time - repair interval + initial delay)
 * Next repair = (start time + repair delay)
 
-Given the formulas above and *start time = 2023-08-25 10:00:00, repair interval = 7 days* the calculation looks like this:
+Given the formulas above and *start time = 2023-08-25 10:00:00, repair interval = 7 days*, initial delay = 1 days (default value) the calculation looks like this:
+* Initial delay = 1 day (default value)
 * Repair delay in days = 1 day
-* Completed at = 2023-08-19 10:00:00
+* Completed at = 2023-08-25 10:00:00
```

Comment:
This is wrong. "starttime - interval + initial delay" in this case still equals 2023-08-19 (25 - 7 + 1)